frontend/src/store/wallet.ts

```diff
@@ -1,4 +1,4 @@
-import { computed, onMounted, ref, watch } from 'vue';
+import { computed, onMounted, markRaw, ref, watch } from 'vue';
```

Comment:
_(arbitrary comment location to start a thread)_

> Note: I think there's one more place we need makeRaw -- namely [here](https://github.com/ScopeLift/umbra-protocol/blob/1fe4ef016b687d6b6c45e7487e41abca105c5e59/frontend/src/store/wallet.ts#L344-L345) -- but I haven't been able to verify yet

About to look into this, will resolve this conversation afterwards

Reply:
Ok, I was able to execute the linked code successfully without `markRaw`, so I guess we don't need it there. Let's merge this PR as the only `markRaw` for now, then add more as-needed when that error pops up. Leaving this comment unresolved so @davidlaprade sees it